docs/Documentation/Applications/comsol.md

```diff
@@ -7,57 +7,72 @@ parent: Applications
 
 *COMSOL Multiphysics is a versatile finite element analysis and simulation package. The COMSOL graphical user interface (GUI) environment is supported primarily for building and solving small models while operation in batch mode allows users to scale their models to larger, higher-fidelity studies.*
 
-Currently, we host three floating network licenses and a number of additional modules. Issue the command `lmstat.comsol` to see current license status and COMSOL modules available.
+Currently, we host three floating network licenses and a number of additional modules. Issue the command `lmstat.comsol` to see current license status and COMSOL modules availability.
```

Comment:
We shouldn't direct users to issue the lmstat.comsol command before explaining how to set up the script to make that command work. Since we explain below how to set up the lmstat.comsol command and how to use it to check license status, I think the line here ("Issue the command `lmstat.comsol` to see current license status and COMSOL modules availability.") can just be removed.